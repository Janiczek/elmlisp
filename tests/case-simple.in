(case bool
  (True 1)
  (False 0))
