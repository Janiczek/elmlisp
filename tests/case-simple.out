case bool of
    True ->
        1

    False ->
        0
